LEAF_Request_Portal/sources/IconPIcker.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/*
+ * As a work of the United States government, this project is in the public domain within the United States.
+ */
+
+/*
+    System controls
+    Date Created: September 17, 2015
+
+*/
+
+$currDir = dirname(__FILE__);
+
+include_once $currDir . '/../globals.php';
+
+if (!class_exists('XSSHelpers'))
+{
+    require_once dirname(__FILE__) . '/../../libs/php-commons/XSSHelpers.php';
+}
+if (!class_exists('CommonConfig'))
+{
+    require_once dirname(__FILE__) . '/../../libs/php-commons/CommonConfig.php';
+}
+
+if(!class_exists('DataActionLogger'))
+{
+    require_once dirname(__FILE__) . '/../../libs/logger/dataActionLogger.php';
+}
+
+class IconPicker 

```

Comment:
Coding standards for classes should apply here, return types and typed variables.